IOSAccessAssessment/Views/ContentView.swift

```diff
@@ -41,51 +41,65 @@ struct ContentView: View {
-                    } label: {
-                        Image(systemName: "camera.circle.fill")
-                            .resizable()
-                            .frame(width: 60, height: 60)
-                            .foregroundColor(.white)
-                    }
+        VStack {
+            if manager?.dataAvailable ?? false{
+                ZStack {
+                    HostedCameraViewController(session: manager!.controller.captureSession)
+                    HostedSegmentationViewController(sharedImageData: sharedImageData, selection: Array(selection), classes: Constants.ClassConstants.classes)
                 }
-                else {
-                    VStack {
-                        SpinnerView()
-                        Text("Camera settings in progress")
-                            .padding(.top, 20)
+                Button {
+                    objectLocation.setLocationAndHeading()
+                    manager?.stopStream()
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 2.0) {
+                        navigateToAnnotationView = true
                     }
+                } label: {
+                    Image(systemName: "camera.circle.fill")
+                        .resizable()
+                        .frame(width: 60, height: 60)
+                        .foregroundColor(.white)
                 }
             }
-            .navigationDestination(isPresented: $navigateToAnnotationView) {
-                AnnotationView(sharedImageData: sharedImageData,
-                               objectLocation: objectLocation,
-                               selection: sharedImageData.segmentedIndices,
-                               classes: Constants.ClassConstants.classes,
-                               selectedClassesIndices: selection
-                )
-            }
-            .navigationBarTitle("Camera View", displayMode: .inline)
-            .onAppear {
-                if (manager == nil) {
-                    manager = CameraManager(sharedImageData: sharedImageData)
-                } else {
-                    manager?.resumeStream()
+            else {
+                VStack {
+                    SpinnerView()
+                    Text("Camera settings in progress")
+                        .padding(.top, 20)
                 }
             }
-            .onDisappear {
-                manager?.stopStream()
+        }
+        .navigationDestination(isPresented: $navigateToAnnotationView) {
+            AnnotationView(sharedImageData: sharedImageData,
+                           objectLocation: objectLocation,
+                           selection: sharedImageData.segmentedIndices,
+                           classes: Constants.ClassConstants.classes,
+                           selectedClassesIndices: selection
+            )
+        }
+        .navigationBarTitle("Camera View", displayMode: .inline)
+        .onAppear {
+            openChangeset()
```

Comment:
Just to confirm, this is the only change that has been added to openChangeset right? 
Kinda confused due to the spacing changes.

Reply:
If the change set creation fails, then I am not sure we should be proceeding with the ContentView recording since it will be for nothing. 
Let's see what kind of response we can decide upon if that happens. 
I am not a fan of crashing the application, but we will have to have a retry option.